### DISCUSSION TITLE
Zealot robes should benefit husked players, but not give husk immunity

### DISCUSSION THREAD

**Discussion by NotWendy** [FakeFishGames/Barotrauma#13812] [Ideas and suggestions]
At the moment, zealot robes provide a very powerful effect for a clothes slot, combining the functionality of husk genes with passive healing and making husked enemies neutral to the wearer.

They are also very easily accessible through stealing and can be acquired very early in the campaign, which people generally point at as the biggest problem with the item.

I suggest removing the husk infection immunity stat from the robes, instead making them synergize with husk genes/symbiosis. They'll support husked players by giving them health regeneration, but will still require you to find a way to make husk infection not kill you.

**Comment by JeroonBosch** [comment]
> They are also very easily accessible through stealing and can be acquired very early in the campaign

Can you elaborate on this? Are they appearing in containers for stealing, as at least they aren't defined as loot and shouldn't randomly appear in outposts.

The abandonedoutposthusks mission (Abandoned Outpost) can occur quite early in the campaign, giving an early Zealot Robe, and husk cultist merchants on outposts are wearing a Zealot Robe, but other than that - are there other ways to obtain this early?

**Reply by NotWendy** [reply]
Some members of the church spawn with zealot robes at outposts, as you said. That's all you need. Stun the guy, grab the robes, and get out. At the cost of a small reputation loss you can now ignore husk infection, pressure, oxygen, and have regeneration on top of all that.

**Reply by SomeRandomNoobKekeke** [reply]
Just weld the door and flood them,  you get a robe from merchant and a robe + europan handshake from ecclesiast

**Reply by NotWendy** [reply]
Flooding is a whole another topic, since that strategy breaks everything about the outposts. But yea, you're right. Either way, it is easy to get.

**Comment by JeroonBosch** [comment]
This should now be addressed, on unstable it still gives the healing, but husk immunity has been removed from the robes.

**Comment by Jasontti** [comment]
Tested changes.
Closing discussion as resolved
